src/rust/graph-merger/src/main.rs

```diff
@@ -564,7 +561,7 @@ where
             }
         };
 
-        //        identities.into_iter().for_each(|identity| completed.add_identity(identity));
+        // identities.into_iter().for_each(|identity| completed.add_identity(identity));
```

Comment:
should this just be removed?

Reply:
I can remove it and open a ticket to track that instead.